doc/api_queries.rst

```diff
@@ -66,8 +66,10 @@
     reponame     string     match against packages repositories' names
     version      string     match against packages' versions
     upgrades     boolean    see :meth:`upgrades`. Defaults to ``False``.
+    obsoletes    Query      match packages that obsolete any package from query
     provides     string     match against packages' provides
-    ==========   ========== ===============================================
+    provides     [Reldep]   match against list of Reldeps
```

Comment:
we don't have any `RelDep` documented. It should be documented here or threat it as a string. And instead of duplicities of keys (list of strings or string) I would suggest adding star or something after all keys which values can take a list of the value types supported i.e. `provides*`. Then you can move this line to `In case of value in form of list or query, the filter behaves cumulatively (logical OR).` to star explanation paragraph. Please fix it, rebase it and merge it. Thanks.